src/main/resources/caret-classification-morphtest.template

```diff
@@ -0,0 +1 @@
+# metamorphic tests are not yet implemented for Caret framework
```

Comment:
Would atoml crash without these files? if there is no crash, they should be deleted. If there is a crash, a good exception would still be better (see above)